tests/test_chathistory_mongo.sh

```diff
@@ -0,0 +1,69 @@
+﻿#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+
+WORKPATH=$(dirname "$PWD")
+ip_address=$(hostname -I | awk '{print $1}')
+
+export MONGO_HOST=${ip_address}
+export MONGO_PORT=27017
+export DB_NAME=${DB_NAME:-"Conversations"}
+export COLLECTION_NAME=${COLLECTION_NAME:-"test"}
+
+function build_docker_images() {
+    cd $WORKPATH
+    echo $(pwd)
+    docker run -d -p 27017:27017 --name=mongo mongo:latest
```

Comment:
`--name=xxx` -> `--name=test-comps-xxx`  
We use this prefix for GenAIComps test containers, to make it separate with GenAIExamples CI test. We have limit HW resource, so we should make sure the name won't be conflict.

Reply:
Done